### ISSUE TITLE
Incremental throws NotFittedError while scoring after fitting when using distribtued scheduler

### ISSUE THREAD

**Issue by stsievert** [dask/dask-ml#271]
Everything is fine when I create an `Incremental` model before calling `fit` then `score` and only use dask arrays. However, when I create a distributed scheduler and call `Incremental.fit` before `Incremental.score`, a `NotFittedError` is raised.

It looks like this involves #258  (the example below passes when I checkout 8acf604c61c2459ead0b23dfac2c3ae33f550922).

Example:

``` python
from dask_ml.datasets import make_classification
import dask.array as da
from sklearn.linear_model import SGDClassifier
from dask_ml.wrappers import Incremental
from distributed import Client

# commented: no errors. uncommented: NotFittedError for inc.core.
client = Client()

X, y = make_classification(random_state=0, chunks=20)
classes = da.unique(y).compute()

est = SGDClassifier(loss='log', penalty='l2', tol=0e-3, average=True)
inc = Incremental(est, scoring='accuracy')
inc.partial_fit(X, y, classes=classes)
inc.score(X, y)
```

**Comment by TomAugspurger** [comment]
I was just writing up this issue :)

The surface problem is https://github.com/dask/dask-ml/blob/d5801584d092d8f13f1b38aaf4da5dc3caa6a213/dask_ml/wrappers.py#L364

The `result` is not `estimator` there.

Setting `self.estimator_ = result` is fine here, since we create `estimator` in that function.

**Comment by TomAugspurger** [comment]
I haven't been able to reproduce with a unit test yet. Working on it now.

**Comment by stsievert** [comment]
> I was just writing up this issue :)

Good to hear – I was worried that this wasn't actually an issue because I couldn't reproduce in a test.
